meta/meta.go

```diff
@@ -488,6 +489,54 @@ func (m *Meta) CreateTableOrView(dbID int64, tableInfo *model.TableInfo) error {
 	return m.txn.HSet(dbKey, tableKey, data)
 }
 
+// SetDDLTables write a key into storage.
+func (m *Meta) SetDDLTables() error {
+	_, err := m.txn.Inc(mInitDDLTable, 1)
+	return errors.Trace(err)
+}
+
+// CreateMySQLSchema create mysql schema
+func (m *Meta) CreateMySQLSchema() (int64, error) {
+	dbs, err := m.ListDatabases()
+	if err != nil {
+		return 0, err
+	}
+	if len(dbs) != 0 {
```

Comment:
this ```if``` is useless. ```range``` will check whether the slice is empty or not.

Reply:
update in the newest commits, PTAL